src/Configuration/src/Abstractions/BaseServiceOptions.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Steeltoe.Common;
+using Steeltoe.Common.Options;
+
+namespace Steeltoe.Configuration;
+
+/// <summary>
+/// Represents a service in ASP.NET configuration, containing nested services grouped by type.
+/// </summary>
+/// <remarks>
+/// Binds with an <see cref="IConfiguration" /> when instantiated.
+/// </remarks>
+public abstract class BaseServiceOptions : AbstractOptions
```

Comment:
Should `AbstractOptions` be renamed too?

Reply:
I didn't because it is not defined in the Configuration namespace. Do you want me to?

Reply:
I figured that might be the case... Let's leave it for now